tests/__tests__/refmterr/refmterr_test.re

```diff
@@ -160,6 +159,9 @@ let forEachTest =
       | "Win32" => "| sed -E \"s/([A-Za-z0-9])\\\\\\\\([A-Za-z0-9])/\\\\1\\\\/\\\\2/g\" | sed -E \"s/\\\\r//g\""
       | _ => ""
       };
+
+    Pastel.(setMode(Terminal));
```

Comment:
I think if we use this we actually want to use Pastel.(useMode()) instead of setMode because setMode basically modifies global state and isn't intended to be used in situations like this.

I think this change isn't being propogated through (e.g. the ANSI disappeared from the snapshots) because we are kicking off a separate .exe. I imagine that adding FORCE_COLOR=true to the command below will get the effect that we want (or at least what we had before).

Reply:
Hey @bandersongit, I wanted to make it with `useMode` but the idea behind useMode makes only limited one call of Pastel to set a mode and restore it afterwards (If I understand it correctly from here https://reason-native.com/docs/pastel/api#modes)

I could cache the mode before the tests and restore it back at the end of the execution. Does it make sense? 

I will add `FORCE_COLOR=true` into all the cmds of those tests. Thanks!

Reply:
>I could cache the mode before the tests and restore it back at the end of the execution. Does it make sense?

That's exactly what useMode would do if you wrapped the body of the test in useMode, however because we are creating a separate process I don't think it would work. If you are saying do it as a "beforeAll/afterAll", we would need to wait for the lifecycle methods you mentioned because of how discovery works (e.g. all describes are run before any tests are run as part of test discovery)

Either way the Pastel call here should be deleted